R/run-url.R

```diff
@@ -43,8 +44,14 @@ runUrl <- function(url, filetype = NULL, subdir = NULL, ...) {
     stop("Unknown file extension.")
 
   message("Downloading ", url)
-  filePath <- tempfile('shinyapp', fileext=fileext)
-  fileDir  <- tempfile('shinyapp')
+  if (is.null(destDir)) {
+    filePath <- tempfile('shinyapp', fileext = fileext)
+    fileDir  <- tempfile('shinyapp')
+  } else {
+    fileDir <- destDir
+    filePath <- paste(destDir, fileext)
+  }
+  message("App data files in ", fileDir)
```

Comment:
I don't think it's necessary to have a `message()` here, since the user presumably would know that they've specified a destination directory.